.gitignore

```diff
@@ -130,3 +130,8 @@ dmypy.json
 
 # Project specific
 configurable_http_proxy/version.txt
+/.idea/configurable-http-proxy.iml
```

Comment:
We should just make a separate section for `# Editors` and ignore the entire `/.idea` folder